qless.ru

```diff
@@ -1,7 +1,8 @@
 require 'qless'
 require 'qless/server'
 
-client = Qless::Client.new(:host => ENV['REDIS_HOST'], :port => ENV['REDIS_PORT'].to_i, :db => ENV['DB_NUM'].to_i )
+# use REDIS_URL="redis://some-host:7000/3"
```

Comment:
Update the `README.md` to specify it is using `REDIS_URL` instead of these other variables.